README.md

```diff
@@ -1,63 +1,51 @@
 # Screwdriver Jenkins Executor
 [![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][status-image]][status-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
 
-> This executor plugin extends the [executor-base-class], and provides methods to start jobs from Jenkins
+> Jenkins Executor plugin for Screwdriver
 
 ## Usage
 
 ```bash
-npm install screwdriver-executor-j5s
+npm install screwdriver-executor-jenkins
 ```
 
 ### Configure
 The class provides a couple options that are configurable in the instantiation of this Executor
 
-| Parameter        | Type  |  Description |
-| :-------------   | :---- | :-------------|
-| config        | Object | Configuration Object |
-| config.username | String | The username for Jenkins cluster  |
-| config.password | String | The password or token for Jenkins cluster  |
-| config.host | String | The hostname for the Jenkins cluster |
+| Parameter        | Type  | Default    | Description |
+| :-------------   | :---- | :----------| :-----------|
+| config        | Object | | Configuration Object |
+| config.ecosystem | Object | | Screwdriver Ecosystem (ui, api, store, etc.) |
```

Comment:
Because all other executors (k8s, k8s-vm, docker) have the same `config.ecosystem` description, I think it is better to change them all at once with other PRs, if possible...